### PR TITLE
Grenade Launchers & Flare Guns now go "Bloop!"

### PR DESCRIPTION
**Pull request by Laniakeaaa**
Grenade Launchers, Flare Guns and Flash Shells all now make a iconic bloop sound when fired.
Mixed Shotgun and Bloop together for Flash Shell sounds.
Load & Unload Sounds.

**Sound Credits:**

https://freesound.org/people/LeMudCrab/sounds/163458/ - (grenade_launcher_fire.ogg, shotgun_fire_flare.ogg)

https://freesound.org/people/PNMCarrieRailfan/sounds/681676/ - (grenade_launcher_load.ogg, grenade_launcher_unload.ogg)

### CODE REVIEW

**Review by ggermanicus — APPROVED** [review]

**Review by CPU-Blanc — CHANGES_REQUESTED** [review]
The second sound is licensed under CC BY 4.0. Although crediting on GitHub _might_ be sufficient, putting it in the codebase itself is a little more solid for adhering to the license terms.

Just add the credits to `sound/urist/Credits.txt` and this is good to go

**Comment by Laniakeaaa** [comment]
Good spot, thanks. I've updated the credits now

File: sound/urist/Credits.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 buzzclick.ogg - Made by BockelSound (https://freesound.org/people/BockelSound/sounds/489503/)
+grenade_launcher_load.ogg, grenade_launcher_unload.ogg - Made by PNMCarrieRailfan (https://freesound.org/people/PNMCarrieRailfan/sounds/681676/)
 
 All sounds above this line have been used under the Creative Commons Attribution 3.0 (CC BY 3.0) license: https://creativecommons.org/licenses/by/3.0/
```
Comment (CPU-Blanc) [inline sound/urist/Credits.txt:4]: I was going to poke a slight nitpick that the license for this particular sound is CC BY **4.0** and not 3.0, however, looks like the license for the other sound has actually been updated to 4.0, so you can just update the disclaimer.

This is _technically_ also required in the CC BY 4.0 license (see [here](https://creativecommons.org/licenses/by/4.0/deed.en#ref-appropriate-credit)).
```suggestion
All sounds above this line have been used under the Creative Commons Attribution 4.0 (CC BY 4.0) license: https://creativecommons.org/licenses/by/4.0/
```

Comment (Laniakeaaa) [reply sound/urist/Credits.txt]: should be good now

**Review by CPU-Blanc — APPROVED** [review]

**Pull request merged**